docs/security/user-certs.md

```diff
@@ -10,13 +10,23 @@ Getting a User Certificate
 --------------------------
 
 This section describes how to get and set up a personal certificate to use on OSG.
-Currently you can get a certificate from either the OSG CA or from CILogon.
-After May 31st, you will only be able to get a certificate issued by CILogon.
-Other CAs may be used; if your virtual organization (VO) requires that you get a certificate from a different CA, [contact your VO Support Center](http://www.opensciencegrid.org/?pid=1000187) for instructions.
+You need an user certificate if you are going to interact directly with OSG resources or infrastructure. 
+Examples of this would be tasks like managing OASIS, directly running jobs on OSG resources, interactingly directly with 
+OSG storage elements, or updating or querying OSG site information (such as site topology).
+Currently, you can get am user certificate from CILogon.
+You may also be able to user other CAs to get a certificate; if your virtual organization (VO) requires that you get a
+certificate from a different CA, [contact your VO Support Center](https://github.com/opensciencegrid/topology/tree/master/virtual-organizations) for
+instructions.
 
 ### Know your responsibilities
 
-When you request a certificate your provide some public personal information about yourself; *name, email address, phone number*, and which *VO* (virtual organization) you belong to. If any of this information changes after you have your certificate you should notify OSG (and probably VO) of the changes. For the OSG RA send email to osg-ra at opensciencegrid dot org.
+When you receive an user certificate, 
+When you request a certificate, you provide some public personal information about yourself; *name, email address, phone
+number*, and which *VO* (virtual organization) you belong to. 
+If any of this information changes after you have your certificate you should notify CILogon (and probably VO) of the changes. 
```

Comment:
You don't need to provide any of the above in the new cilogon world order. The subsequent sentence can also be removed

Reply:
done